### PR TITLE
Increase code coverage

### PR DESCRIPTION
**Pull request by jean-michelet**
Related to issue #216 

I think an error has been introduced by https://github.com/fastify/under-pressure/pull/182, `test.js` is not executed since 2022 :smiling_face_with_tear: 

I renamed it `index.test.js`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -304,10 +304,12 @@ async function fastifyUnderPressure (fastify, opts) {
   }
 }
 
+/* c8 ignore start */
```
Comment (Uzlopak) [inline index.js:307]: So many c8 ignore. Why not simply didable coverage all together?

Comment (jean-michelet) [reply index.js:307]: How would you do it?
If I understand correctly, it's related to conditions with `monitorEventLoopDelay`.

**Review by gurgunday — APPROVED** [review]
lgtm

**Comment by gurgunday** [comment]
I’m personally fine with both the ignores and disabling directly

File: index.js (hunk 1 above)
Comment (gurgunday) [reply index.js:307]: He is saying just leave disable-coverage: true if there are too many lines uncovered

Comment (jean-michelet) [reply index.js:307]: Ho, I misunderstood "altogether".

I think ignoring lines that can't be evaluated by CI for well documented reasons is better from completely disabling the coverage. In the first case, you're still forcing developers to think about why some lines aren't covered. 

Is the increase in the number of lines of code the problem?

File: index.js [hunk 2]
```diff
@@ -16,14 +16,14 @@ const TYPE_HEALTH_CHECK = 'healthCheck'
 const TYPE_EVENT_LOOP_UTILIZATION = 'eventLoopUtilization'
 
 function getSampleInterval (value, eventLoopResolution) {
+/* c8 ignore start - @see: https://github.com/fastify/under-pressure/issues/18 */
```
Comment (mcollina) [inline index.js:19]:
```suggestion
  /* c8 ignore start - @see: https://github.com/fastify/under-pressure/issues/18 */
```

**Review by mcollina — CHANGES_REQUESTED** [review]
I think those branches are just not runnable anymore in the fastify v5 line/with the support matrix we have.

**Comment by mcollina** [comment]
I think you can just remove those lines.

**Comment by jean-michelet** [comment]
Ok, I am working on it

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by Uzlopak** [comment]
@jean-michelet 
Great job ;).

**Comment by jean-michelet** [comment]
:pray:

**Pull request merged**